lib/msf/core/exploit/remote/ldap.rb

```diff
@@ -135,5 +135,29 @@ def discover_base_dn(ldap)
       print_good("#{peer} Discovered base DN: #{base_dn}")
       base_dn
     end
+
+    def fail_unless_bind_succeeded(ldap)
```

Comment:
not a blocker:

```suggestion
    def validate_bind_success!(ldap)
```

Context: The bang method naming convention helps show intent to the reader that the method can raise as as side-effect:

```ruby
      ldap_connect do |ldap|
          validate_bind_success!(ldap)
```